src/parser/ast/ReturnStatementNode.h

```diff
@@ -36,22 +36,14 @@ class ReturnStatementNode : public StatementNode {
     virtual void generateStatementByteCode(ByteCodeBlock* codeBlock, ByteCodeGenerateContext* context) override
     {
 #ifdef ESCARGOT_DEBUGGER
-        if (context->m_breakpointContext->m_breakpointLocations->size() == 0 && context->m_breakpointContext->m_parsingEnabled) {
-            ASSERT(context->m_breakpointContext->m_lastBreakpointLineOffset == 0);
-            ASSERT(context->m_breakpointContext->m_breakpointLocations->size() == 0);
-
+        {
```

Comment:
Do we need this `{}` block?

Reply:
No, it's just a matter of taste.
I want to seperate debugger-relate part clearly from other code.